chrome/test/sunset-test.js

```diff
@@ -45,3 +45,40 @@ test(function () {
   current.setSeconds(current.getSeconds() - 1);
   assert_false(Sunset.shouldShowNotification_(start, current, 1));
 }, "Edge-cases for shouldShowNotification_");
+
+test(function () {
```

Comment:
Nit: This really ought to be an `async_test` with `step_func`s for each callback, but I suppose it makes no difference given how you've structured it (and given that the callbacks aren't really async in the mock).